README.md

```diff
@@ -129,4 +129,19 @@ Bump versions and publish with:
 npm publish --access public
 ```
 
+## Ignore Files
+
+Create a file called `.claspignore` in the root directory of your Apps Script project. Add files as if it were a .gitignore, and they will be excluded from `clasp push`, `clasp deploy`, etc...  
+
+As of now, clasp does not support pushing only tracked files, so to push a few files from an npm project with bundling, for example, a sample `.claspignore` could look like:
```

Comment:
> As of now, clasp does not support pushing only tracked files, so to push a few files from an npm project with bundling, for example, a sample `.claspignore` could look like:

I don't think `clasp` will ever support tracked files like git. Let's just say:

"""
A sample `.claspignore` could look like:
"""

The next sentence explains the file.